drivers/scheduler/k8s/k8s.go

```diff
@@ -2385,6 +2385,76 @@ func (k *K8s) GetVolumes(ctx *scheduler.Context) ([]*volume.Volume, error) {
 	return vols, nil
 }
 
+// GetPureVolumes  Get Pure FB Volumes
+func (k *K8s) GetPureVolumes(ctx *scheduler.Context) ([]*volume.Volume, error) {
+	k8sOps := k8sApps
+	var vols []*volume.Volume
+	for _, specObj := range ctx.App.SpecList {
+		if obj, ok := specObj.(*corev1.PersistentVolumeClaim); ok {
+			pvcObj, err := k8sCore.GetPersistentVolumeClaim(obj.Name, obj.Namespace)
+			if err != nil {
+				return nil, err
+			}
+			scForPvc, err := k8sCore.GetStorageClassForPVC(pvcObj)
+			if err != nil {
+				return nil, err
+			}
+			fmt.Println("printing scforpvc params: " + scForPvc.Parameters["backend"])
```

Comment:
is this a debugging message? if not we may need to log it properly.

Reply:
Agreed, this should be a log call or removed